x-pack/plugins/actions/server/saved_objects/transform_connectors_for_export.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObject } from 'kibana/server';
+import { RawAction } from '../types';
+
+const CONNECTORS_WITHOUT_SECRETS = ['.index', '.server-log'];
```

Comment:
Maybe here we can replace the constants with the dynamic approach to check the action type schema?

Reply:
@YulNaumenko Great idea! I updated this in this commit https://github.com/elastic/kibana/pull/98802/commits/58dfb8ab69ee989c182c79a2a3fb2e816d521556

I ran into the issue where the email and webhook secrets validation does not take into account whether `hasAuth` is set to `false` inside the connector parameters so in this case, if the secrets validation fails, I'm still checking for the existence of `hasAuth`. Do you think that works?